graders/java/autograder.sh

```diff
@@ -63,7 +63,7 @@ chmod 777 /grade/params/params.json
 # Disable Java management options to hinder student's ability to dump
 # the heap. The port is not a valid int on purpose, to trigger an
```

Comment:
Should the comment here about the port be removed? (I wonder if the incidents reported on Slack related to this were students trying to do something...)

Reply:
It was not. The grader was failing even with regular submissions (it was pretty easy to reproduce). But yes, the comment can go. PR has been merged for now, but I'll likely revisit this and make the protection stronger when I have some time.